### DISCUSSION TITLE
Does it work with 'newly' ESP32-S3 hardware?

### DISCUSSION THREAD

**Discussion by giangy11** [syssi/esphome-jk-bms#393] [General]
Hi,
is this module also working with an ESP32-S3 version?
Edit: yes it works!

**Comment by testpaul999** [comment]
> Hi, is this module also working with an ESP32-S3 version? Edit: yes it works!

Can you say pls what you use at this line
esp32:
  board: _wemos_d1_mini32_


Regards

**Reply by giangy11** [reply]
![Screenshot_20231214_185309_Home Assistant](https://github.com/syssi/esphome-jk-bms/assets/6941069/804e4348-9eed-48dc-95d9-885f8efebe22)

**Reply by testpaul999** [reply]
Thx

**Reply by testpaul999** [reply]
[compile fehler.txt](https://github.com/syssi/esphome-jk-bms/files/13678307/compile.fehler.txt)

don't work here
Esphome 2023.5.5

**Reply by syssi** [reply]
Using the most recent ESPHome release (`2023.11`) is probably important because the board/support is pretty new.

**Reply by testpaul999** [reply]
Thx build runs now.
